src/appleseed-max-impl/appleseedoslplugin/oslparamdlg.cpp

```diff
@@ -175,14 +265,18 @@ void OSLParamDlg::add_ui_parameter(
     const int EditHeight = 10;
     const int TexButtonWidth = 84;
 
+    int col_1x = 10;
```

Comment:
May I suggest `col1_x` (and then use this convention everywhere)? This would make it clearer that it's the X position of col1, because 1x means "one time" to me...